modules/demo/spatial/app.js

```diff
@@ -198,9 +198,9 @@ function copyPointsDtoH(polyIds, points) {
 
   const positions = (() => {
     const size = points.numRows;
-    const positions = Series.sequence({ type: new Float32, init: 0, size: size * 3, step: 0 });
-    positions.scatter(points.get('x'), Series.sequence({ type: new Uint32, init: 0, size, step: 3 }));
-    positions.scatter(points.get('y'), Series.sequence({ type: new Uint32, init: 1, size, step: 3 }));
+    const positions = Series.sequence({ type: new Float32, init: 0, size: size * 3, step: 0 })
+      .scatter(points.get('x'), Series.sequence({ type: new Uint32, init: 0, size, step: 3 }))
+      .scatter(points.get('y'), Series.sequence({ type: new Uint32, init: 1, size, step: 3 }));
```

Comment:
removed chained calls

Reply:
Well, my in retrospect my comment was a bit oblique. I meant as I user I think I'd like a mutating version.

Reply:
I understand that for sure. The thought behind this was to be consistent across functions to not be inplace. May be since we already have a couple of `InPlace` functions now, we can have a version `scatterInPlace`, and make that a standard for all inplace functions?

Reply:
I generally favor immutability both because it's safer for end-users, and aligns with the C++ APIs.

I also want to avoid the bifurcation in the Python APIs where most ops are immutable, but some have an "in-place" variant/option. IMO having multiple ways to do the a thing is fine for intermediate/advanced users, but more confusing for newcomers.

`Series.fill`/`Series.fillInPlace` is at least partial precedent for having both. My argument against it here is that `libcudf` has a C++ `cudf::fill_in_place` method, but no `cudf::scatter_in_place`.

Reply:
That being said, we _essentially_ already have `scatterInPlace` in the form of `setValue`, albeit for single index/value pairs. `Series.setValue` working in-place is probably good to keep in order to match the basic semantics of other "array-like" types.

IMO extending this further to also have a `setValues(indices, values)` method seems like a good compromise. @AjayThorve @bryevdv thoughts?

Reply:
agreed with the `setValues` solution

Reply:
Also sounds good to me!

Reply:
done